src/utils.ts

```diff
@@ -76,8 +78,11 @@ const sleep = (ms) => {
 
 export const waitTillAvailable = async (url: string, opts?: WaitTillAvailableOptions): Promise<void> => {
   const options: WaitTillAvailableOptions = { ...defaultOptions, ...opts }
+  if (env.isDev) options.confirmations = 1
   const isHttps = url.startsWith('https://')
-  const rejectUnauthorized = !(options.skipSsl || !process.env.NODE_TLS_REJECT_UNAUTHORIZED)
+  const globalSkipSsl = !env.NODE_TLS_REJECT_UNAUTHORIZED
```

Comment:
It is hard for me to find a use case for logic around TLS flags.
Is environment instructs to skip TLS validation (NODE_TLS_REJECT_UNAUTHORIZED) then why a user would like to enforce it. And the other way round. Could you please clarify your understanding of this issue?

Reply:
good point...it was just another version of what marc made but I guess it serves no use...wil remove